bluemira/base/config.py

```diff
@@ -135,6 +147,17 @@ class Configuration(ConfigurationSchema, ParameterFrame):
         ["f_lfs_upper_target", "Fraction of SOL power deposited on the LFS upper target (DN only)", 0.5, "dimensionless", None, "Input"],
         ["f_hfs_upper_target", "Fraction of SOL power deposited on the HFS upper target (DN only)", 0.5, "dimensionless", None, "Input"],
 
+        # Variables for radiation
+        ["p_sol", "power entering the SoL", 300e6, "W", None, "Input"],
```

Comment:
There is for sure some duplication of parameters here... and something like `gamma` is certainly to generic a name for something so specific

Reply:
I changed `p_sol` with the already existing` P_rad`, and I renamed `gamma` with `gamma_sheath`. Together with the description it should be quite explicit